data/boarding missions.txt

```diff
@@ -8,32 +8,335 @@
 # WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 # PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
-mission "Assisting Merchant"
+conversation "assisting merchant"
+	`When you repair the <origin>, the captain thanks you for your assistance and pays you <payment>.`
+		decline
+
+mission "Assisting Merchant (Small)"
 	assisting
 	repeat
+	minor
 	to offer
-		random < 10
+		random < 20
 	source
-		government Merchant
+		government "Merchant"
+		category "Interceptor" "Light Warship" "Transport" "Light Freighter"
 	on offer
 		payment 10000
-		conversation
-			`When you repair the <origin>, the captain thanks you for your assistance and pays you <payment>.`
-				decline
+		conversation "assisting merchant"
 	# A mission with no destination will not be offered, so give it one.
 	# What the destination is doesn't matter, because you always 'decline' this.
-	destination Earth
+	destination "Earth"
+
+mission "Assisting Merchant (Medium)"
+	assisting
+	repeat
+	minor
+	to offer
+		random < 15
+	source
+		government "Merchant"
+		category "Medium Warship" "Heavy Freighter"
+	on offer
+		payment 20000
+		conversation "assisting merchant"
+	destination "Earth"
+
+mission "Assisting Merchant (Large)"
+	assisting
+	repeat
+	minor
+	to offer
+		random < 10
+	source
+		government "Merchant"
+		category "Heavy Warship"
+	on offer
+		payment 30000
+		conversation "assisting merchant"
+	destination "Earth"
+
+
+
+mission "Dangerous Journey"
+	assisting
+	minor
+	name "Delivery to <planet>"
+	description "Deliver <cargo> to <destination>. Payment is <payment>."
+	to offer
+		random < 10
+	source
+		government "Merchant"
+		category "Light Freighter" "Heavy Freighter"
+	destination
+		government "Republic" "Syndicate" "Free Worlds"
+		attributes "frontier"
+		distance 2 3
+	cargo random 5 2 .1
+	on offer
+		conversation
+			`When you repair the <origin>, the captain hails you. "This is the second time I've nearly died in the past week doing this job! Here, take this <cargo> to <destination> and finish this job for me. They said the payment is <payment>."`
```

Comment:
I get that the captain is upset, but this is an awfully rude and hostile way to treat someone who's just saved your life! Maybe tone down the anger a bit?

Reply:
Not everyone in the galaxy is calm tempered and nice. Really, this guy’s just taking his anger out on you. (Originally when I wrote down the idea for this mission, I said that the captain had been disabled three times prior. I ended up writing down two, but I can definitely increase the number of times mentioned here so that the player can get more of a sense of this captain’s rage.)

Reply:
That might help, or else having him swear generically in addition to or instead of at you specifically.